### ISSUE TITLE
Add parameter for hosting domain

### ISSUE THREAD

**Issue by fivetwelve** [toitoi-co/toitoi-admin#12]
Include parameter in config.json to store hosting domain that is returned in login response user object, possibly within the user.site properties.

This domain is currently hardcoded in -cms api calls to Firebase.

**Comment by joepie91** [comment]
Does this refer to the `.toitoi.co` suffix?

**Comment by fivetwelve** [comment]
Yes, it does. This was something you thought of on Slack and asked me to add an issue for it. :)

**Comment by joepie91** [comment]
Alright, just wanted to clarify, as I wasn't sure about the specifics anymore :)
